### PR TITLE
Add requirements for advancing the maturity level of features

### PR DESCRIPTION
**Pull request by nigelmegitt**
Addresses #66.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/charter-timed-text/pull/67.html" title="Last updated on Oct 15, 2021, 4:56 PM UTC (7ec4ec1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/charter-timed-text/67/935f16c...7ec4ec1.html" title="Last updated on Oct 15, 2021, 4:56 PM UTC (7ec4ec1)">Diff</a>

### CODE REVIEW

**Comment by gkatsev** [comment]
Do we want/need to keep this piece at the beginning of this section with this change?
```html
<p>In order to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
```

**Comment by nigelmegitt** [comment]
Oh good point @gkatsev - will fix. Thanks!

File: index.html [hunk 1]
```diff
@@ -286,7 +286,24 @@ <h2>Success Criteria</h2>
...
+      <li>Presentation implementation</li>
+      <li>Content</li>
+      <li>Validating implementation</li>
+    </ul>
+    <p>For example, a feature MAY be advanced beyond Candidate Recommendation
+    if it has been demonstrated to be implementable on the basis of a single
+    open source implementation that successfully processes content from an
+    independent source.</p>
+    </section>
 <!-- END: need to consider of -->
```
Comment (himorin) [inline index.html]: propose to remove this line also.

**Comment by himorin** [comment]
I confirm format is ok.

**Review by himorin — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (nigelmegitt) [reply index.html]: Yeah I left that in because I wasn't sure where the beginning of the marked section was! Happy to remove.

Comment (nigelmegitt) [reply index.html]: Done.

**Review by gkatsev — APPROVED** [review]

**Comment by himorin** [comment]
@nigelmegitt @gkatsev PR has been opened over two weeks, are we fine as approved, or need to wait for next our regular call?

**Comment by nigelmegitt** [comment]
Happy to merge.

**Pull request merged**